pinot-query-planner/src/main/java/org/apache/pinot/query/planner/plannode/AggregateNode.java

```diff
@@ -42,6 +45,12 @@ public class AggregateNode extends AbstractPlanNode {
   private List<RexExpression> _groupSet;
   @ProtoProperties
   private AggType _aggType;
+  @ProtoProperties
+  private List<RexExpression> _collationKey = ImmutableList.of();
+  @ProtoProperties
+  private List<RelFieldCollation.Direction> _collationDirection = ImmutableList.of();
+  @ProtoProperties
+  private List<RelFieldCollation.NullDirection> _collationNullDirection = ImmutableList.of();
```

Comment:
Do you put them as 3 separate list because we cannot serialize `RelFieldCollation` properly? Some comments will help

Reply:
done